apps/system/js/accessibility.js

```diff
@@ -388,6 +431,14 @@
     speak: function ar_speak(aData, aCallback, aOptions = {}) {
       this.speechSynthesizer.speak(aData, aOptions, this.rate, this.volume,
         aCallback);
+    },
+
+    /**
+     * Cancel any utterances currently being spoken by speechSynthesis.
+     * @memberof Accessibility.prototype
+     */
+    cancelSpeech: function ar_cancelSpeech() {
```

Comment:
Could you replace all other instances where this.speechSynthesizer.cancel(); is used with the new method.